api/android/api/src/main/jni/Android.mk

```diff
@@ -74,7 +74,7 @@ GSTREAMER_NDK_BUILD_PATH := $(GSTREAMER_ROOT)/share/gst-android/ndk-build/
 include $(LOCAL_PATH)/Android-gst-plugins.mk
 
 GSTREAMER_PLUGINS        := $(GST_REQUIRED_PLUGINS)
-GSTREAMER_EXTRA_DEPS     := $(GST_REQUIRED_DEPS) gio-2.0
+GSTREAMER_EXTRA_DEPS     := $(GST_REQUIRED_DEPS) gio-2.0 gmodule-2.0
```

Comment:
Does this remove the direct-dependency on dlfcn/ldl ?

Reply:
I removed dl-dep in nnstreamer.

Reply:
Oh.. then, it's great!